simplistic_editor/lib/basic_text_input_client.dart

```diff
@@ -40,7 +40,7 @@ class BasicTextInputClient extends StatefulWidget {
 }
 
 class BasicTextInputClientState extends State<BasicTextInputClient>
-    with TextSelectionDelegate
+    with TextSelectionDelegate, TextInputClient
```

Comment:
I think we should also mixin the `DeltaTextInputClient` instead of implementing it. This  helps the sample in that every addition to `TextInputClient` will not break the sample. More context here https://github.com/flutter/flutter/issues/114380#issuecomment-1299137406 .

Reply:
Makes sense to me, I'll make that change. Thanks!